code/modules/vore/fluffstuff/guns/dominator.dm

```diff
@@ -1,7 +1,7 @@
 // -------------- Dominator -------------
 /obj/item/weapon/gun/energy/gun/fluff/dominator
-	name = "\improper MWPSB Dominator"
-	desc = "A MWPSB's Dominator from the Federation. Like the basic Energy Gun, this gun has two settings. It is used by the United Federation Public Safety Bureau's Criminal Investigation Division. The weapon can only be fired by the owner and is alert-level locked."
+	name = "bulky energy gun"
```

Comment:
This should be "Federation energy gun" considering only the Federation uses it.

Reply:
Would that be a layman's judgement when seeing the thing?

It's bulky af.